src/OpenTelemetry.AutoInstrumentation/Loading/ILifespanManager.cs

```diff
@@ -25,6 +25,6 @@ internal interface ILifespanManager
     /// it will be disposed together with the manager.
     /// </summary>
     /// <param name="instance">Trackable object</param>
-    void Track(object instance);
+    public void Track(object instance);
```

Comment:
I think that `public` is the only access modifier acceptable inside an `interface`.
I do not recall ever setting it explicitly (at least in this repo).

Reply:
d331a754cc93a64088f40a690b911744384ae2f6